tests/index.html

```diff
@@ -21,6 +21,13 @@
     {{content-for "body"}}
     {{content-for "test-body"}}
 
+    <div id="qunit"></div>
+    <div id="qunit-fixture"></div>
+
+    <div id="ember-testing-container">
```

Comment:
This may not needed, what happens if you remove it?

Reply:
Unsure. Will try.

Reply:
We pulled these changes out into a separate PR - #9

Reply:
Going to push this to a later PR.